consensus/types.go

```diff
@@ -14,15 +24,24 @@ type (
 	Currency    uint64
 	Siafund     uint64
 
+	Identifier [16]byte
+	Signature  []byte
+
 	BlockID     hash.Hash
 	OutputID    hash.Hash
 	ContractID  hash.Hash
-	CoinAddress hash.Hash // The hash of the spend conditions of an output.
+	CoinAddress hash.Hash
 	Target      hash.Hash
 )
 
 var ZeroAddress = CoinAddress{0}
 
+var FileContractIdentifier = Identifier{'f', 'i', 'l', 'e', ' ', 'c', 'o', 'n', 't', 'r', 'a', 'c', 't'}
+var SiacoinOutputIdentifier = Identifier{'s', 'i', 'a', 'c', 'o', 'i', 'n', ' ', 'o', 'u', 't', 'p', 'u', 't'}
+var SiafundOutputIdentifier = Identifier{'s', 'i', 'a', 'f', 'u', 'n', 'd', ' ', 'o', 'u', 't', 'p', 'u', 't'}
+
+var ED25519Identifier = Identifier{'e', 'd', '2', '5', '5', '1', '9'}
```

Comment:
muh verbosity

I'm starting to shift towards the standard library style of "prefixes rather than suffixes" (e.g. ErrFoo). I'd prefer to see these named `IdFileContract`, `Id25519`, etc.